File: app/src/main/java/com/example/mapapp/data/Condition.java
package com.example.mapapp.data;

import org.json.JSONObject;

/**
 * Created by ext-jurkeant on 9/28/2016.
 */

public class Condition implements JSONPopulator {

    private String temp;

    @Override
    public void populate(JSONObject data) {
        temp=data.optString("temp");
    }

    public String getTemp() {
        return temp;
    }
}

File: app/src/main/java/com/example/mapapp/data/Condition.kt
package com.example.mapapp.data

import org.json.JSONObject

/**
 * Created by ext-jurkeant on 9/28/2016.
 */

class Condition : JSONPopulator {

    var temp: String? = null
        private set

    override fun populate(data: JSONObject) {
        temp = data.optString("temp")
    }
}
